src/_pytest/_code/code.py

```diff
@@ -57,6 +56,10 @@
 
 TracebackStyle = Literal["long", "short", "line", "no", "native", "value", "auto"]
 
+EXCEPTION_OR_MORE = Union[Type[Exception], Tuple[Type[Exception], ...]]
+
+type_alias = type  #  to sidestep shadowing
```

Comment:
Is this still needed (also in `src/_pytest/threadexception.py`)? It doesn't seem to actually be used anywhere?

Reply:
remove